### DISCUSSION TITLE
Aggregated feed capped at 400 entries

### DISCUSSION THREAD

**Discussion by haflck** [FreshRSS/FreshRSS#5697] [Q&A] [answered]
Hi everyone.
Today I noticed that if I use the feed agregation ( https://mydomain.tld/i/?a=rss&rid=111111x222222&user=XXXXX&token=XXXXXXXXXXX&hours=168 ) of my main feeds I only get 400 feed entries. It should however be way more. A lot of entries (all after the newest 400) seem to just not be added to the aggregated feed. 

Is it possible to get all entries from the specified timeframe (in this case the last 168hours) in my generated feed - if yes how? Also why is it that way?

**Comment by Alkarex** [comment, marked as answer]
See `max_posts_per_rss` in your `./data/users/alice/config.php` on the model of:

https://github.com/FreshRSS/FreshRSS/blob/573e8e7072b5b91dc158271236de495a85acce32/config-user.default.php#L31

For new users, default values can be overridden with a `./data/config-user.custom.php`.

If you use Docker, see the following link for tips:
https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#modify-the-configuration-of-a-running-freshrss-instance

**Reply by haflck** [reply]
Wow, that was quick! Thanks very much, that's exactly what I needed to know.
